Sources/Workflow/LinkedList/LinkedList.swift

```diff
@@ -53,8 +53,15 @@ public class LinkedList<Value>: Sequence, CustomStringConvertible {
      */
     public var last: Element? { first?.traverseToEnd() }
 
-    /// Creates a `LinkedList` by providing the first node in the list.
+    /// Creates a copy of a `LinkedList` by providing the first node, and copying it.
     public required init(_ node: Element? = nil) {
+        first = node?.copy()
+    }
+
+    /** Creates a `LinkedList` by providing the first node in the list.
+    - Important: This can potentially cause memory retention issues, you are passing a reference, be aware.
+     */
+    public required init(withoutCopying node: Element?) {
```

Comment:
Nice I like this even more than my suggestion.

Reply:
We need to generate the docs for these and other doc changes.  Would you like me to do that before merging or do you want to do it?

Reply:
It’s all you my friend